src/site/markdown/index.md.vm

```diff
@@ -43,3 +43,4 @@ This page provides an overview of the major changes.
 Protobuf-java
 -------------
 Google Protobuf's 3.7.1 jar is available as *org.apache.hadoop.thirdparty:hadoop-shaded-protobuf_3_7* artifact.
```

Comment:
Remove the version from artifact specifier too? Will be odd if you upgrade to pb 2.8 but you specify the dependency referring to hadoop-shaded-protobuf_3_7 ?